server/app/auth/oidc/CustomOidcLogoutRequest.java

```diff
@@ -58,6 +62,21 @@ public CustomOidcLogoutRequest(
     }
   }
 
+  private static JWT getIdToken(ImmutableMap<String, String> extraParams) {
+    if (extraParams == null) return null;
+
+    String idTokenString = extraParams.get("id_token_hint");
+    if (idTokenString == null) {
+      return null;
+    }
+
+    try {
+      return PlainJWT.parse(idTokenString);
```

Comment:
**suggestion (blocking):** please add a test that covers parsing this

Reply:
`toUri_withState` in `CustomOidcLogoutRequestTest` does so - we pass in the entry in the `extraParams` map and then assert that the URL contains the correct ID token.

Do you feel it's necessary to expose this private helper method to tests to test it in isolation, in addition?

Reply:
Ah woops, I misread the codecov annotation in github. Sorry for the noise!